tox.ini

```diff
@@ -96,3 +96,6 @@ changedir = docs
 
 commands =
   sphinx-build -W --keep-going -b html -T . _build/html
+  ; stop jekyll from running on gh-pages
+  ; https://help.github.com/en/articles/files-that-start-with-an-underscore-are-missing
+  touch _build/html/.nojekyll
```

Comment:
Instead of this `touch` we could also use `https://www.sphinx-doc.org/en/master/usage/extensions/githubpages.html`. Makes it easier to build without tox.

Reply:
Good find! Added in 2f3e42f.